app/frontend/src/features/profile/hooks/useRegions.ts

```diff
@@ -0,0 +1,26 @@
+import { useQuery } from "react-query";
+import { service } from "service";
+
+export const useRegions = () => {
+  const { data, ...rest } = useQuery("getRegions", () =>
```

Comment:
```suggestion
  const { data, ...rest } = useQuery("regions", () =>
```
Same comment as in `useLanguages`